### DISCUSSION TITLE
Note aggiuntive in PROFORMA

### DISCUSSION THREAD

**Discussion by stefanosilvestrinimia** [fattureincloud/api#332] [Q&A] [answered]
buonasera, 
vorrei aggiungere note aggiuntive via API che saranno visibili solo nella copia proforma. 

Ho provato a settare: ->setNotes('note') in entity ma non sembra funzionare. Qualche idea?

grazie mille

![Schermata 2023-06-30 alle 16 00 19](https://github.com/fattureincloud/api/assets/114735413/0376bcd3-17aa-4c2d-a6e9-63cf1a37884d)

**Comment by stefanosilvestrinimia** [comment]
hai idee @emajo ? grazie

**Comment by emajo** [comment, marked as answer]
Ciao @stefanosilvestrinimia, è disponibile il campo `data.notes`, settabile con  ->setNotes('note') non nell'entity ma direttamente nel documento.

Grazie,
Emanuele.

**Comment by stefanosilvestrinimia** [comment]
grazie!
